rickAndMortyAPI.test.js

```diff
@@ -0,0 +1,34 @@
+const { getCharacter, getManyCharacters } = require('./rickAndMortyAPI');
+
+describe('getCharacter', () => {
+    it('get character with id = 2', async() => {
+        const result = await getCharacter('https://rickandmortyapi.com/api/character/', 2);
+        const expected =  {
```

Comment:
great work implementing expect.any to make it more broad for this api call.   As you know, last step looks like mocks and you'll be set!